test/e2e/storage/utils/utils.go

```diff
@@ -877,7 +720,90 @@ func VerifyFilePathGidInPod(f *framework.Framework, filePath, expectedGid string
 // ChangeFilePathGidInPod changes the GID of the target filepath.
 func ChangeFilePathGidInPod(f *framework.Framework, filePath, targetGid string, pod *v1.Pod) {
 	cmd := fmt.Sprintf("chgrp %s %s", targetGid, filePath)
-	_, _, err := PodExec(f, pod, cmd)
+	_, _, err := e2evolume.PodExec(f, pod, cmd)
 	framework.ExpectNoError(err)
 	VerifyFilePathGidInPod(f, filePath, targetGid, pod)
 }
+
+// DeleteStorageClass deletes the passed in StorageClass and catches errors other than "Not Found"
+func DeleteStorageClass(cs clientset.Interface, className string) error {
+	err := cs.StorageV1().StorageClasses().Delete(context.TODO(), className, metav1.DeleteOptions{})
+	if err != nil && !apierrors.IsNotFound(err) {
+		return err
+	}
+	return nil
+}
+
+// CreateVolumeSource creates a volume source object
+func CreateVolumeSource(pvcName string, readOnly bool) *v1.VolumeSource {
```

Comment:
@chrishenzie Replying to your concern on the CreateVolumeSource function. I think you are absolutely right. Ideally this function should be a private function in the api package. However, since previously it is under testsuite package so it has been abused by provisioning testsuite:

https://github.com/kubernetes/kubernetes/blob/c970a46bc1bcc100bbbfabd5c12bd4c5d87f8aea/test/e2e/storage/testsuites/provisioning.go#L225

So definitely if we fix provisioning testsuite to not use this, I can put it under storageapi and make it unexported. One thing I noticed when doing this refactor is that a lot of functions have been abused. This PR has been too large to also include those cleanup. So I think it would be good to focus on move the package first. And then we can do clean up for each individual testsuite to make it clean. Does that make sense?